app/assets/stylesheets/application.scss

```diff
@@ -147,3 +147,9 @@ tr.negative {
     transform: scale(1);
   }
 }
+
+.failed-login-banner {
+  background-color: cornsilk;
```

Comment:
Could we use a stronger color like red or orange? I feel like this error message should "pop" a little so it is clearer. What do you think?